docs/tutorials/analysis/1D/spectral_analysis_rad_max.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>I suppose that the following sections are exactly identical to our standard analysis? Maybe helps to explicitly specify so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>But I wrote in the beginning</p><p> </p><p>"Beside the definition of the ON region during the data reduction, the remaining steps are identical to the other&nbsp;<a href="http://localhost:8888/notebooks/spectral_analysis.ipynb" target="_blank">1D spectral analysis example</a>, so we will not detail the data reduction steps already presented in the other tutorial." Is it not clear enough?</p>

Reply:
Yes, data reduction and fitting are probably two distinct steps. There is not differing in the fitting stage...